ui/app/components/clients/current.js

```diff
@@ -1,22 +1,51 @@
 import Component from '@glimmer/component';
-
+import { tracked } from '@glimmer/tracking';
 export default class Current extends Component {
   chartLegend = [
     { key: 'entity_clients', label: 'entity clients' },
     { key: 'non_entity_clients', label: 'non-entity clients' },
   ];
+  @tracked selectedNamespace = null;
+
+  // TODO CMB pass in from parent
+  get upgradeDate() {
+    return this.args.upgradeDate || null;
+  }
+
+  get billingStartDate() {
+    return this.args.billingStartDate || null;
+  }
+
+  // by namespace client count data for partial month
+  get byNamespaceCurrent() {
```

Comment:
Right now we're truncating the payload in the component. We'd like to move this to the serializer and have a separate `Export all data` button to submit a separate request to create a CSV file for _all_ namespace data.

Currently only top 10 namespaces are exported.